docs/reference/field-types/checkboxes.md

```diff
@@ -63,16 +71,68 @@ The `choices` setting in `checkboxes`, `radio`, or `select` fields configures th
 
 ## Populating `choices` dynamically
 
-What if the choices aren't known in advance? Then you can fetch them dynamically.
+What if the choices aren't known in advance or are dependent on the value of another schema field? Then you can fetch them dynamically.
 
-First, set the `choices` option to the name of a [method in your module](../module-api/module-overview.md#methods-self). Pass a string, the name of the method you'll implement on the server side — **do not** pass a function.
+First, set the `choices` option to the name of a [method in your module](../module-api/module-overview.md#methods-self). Pass the name of the method you'll implement on the server side — **do not** pass a function, as a string ending in `()`. e.g. `choices: 'getChoices()'`
```

Comment:
Wording is a little confusing. Maybe reordered:

method you'll implement in the same module, as a string ending in `()` e.g..., **do not** pass a functoin

Reply:
Changed in all three files.